Tests fail for make build-docker

The containerized e2e tests fais due to incorrect volume mounts.
Once the docker image is built, it can be reproduced with ```scripts/e2e-test-docker-containers.sh```.

Output:
```
Running tests -- scripts/e2e-test-docker-containers.sh to verify the binaries
51890559ab50543fd45a2f81f19f4a22475ed9abb7c36c24247f410ec043be0b
2faeecaba98d1909dea20dd92300d0526f61bf9c66b7b301720cdb2c620c5b5c
67e00fd5b9c43020af17846d6b262c74b542619a13cbbf3183b42c46a06add0a
14a2d6fe26df1c4be2c29effbf8fe430c4d267ed0b0cfe3028e2c514c19a7902
312bb34abf3e5a5080fd1dd615006c74fcd35d552cbce71940e95bd8e241c903
21e6b1cab1a4da67c7f3b7daafb99d17320c1ddcd07e3a562fde770c0efcd737
WARN[04-03|04:06:26] Creating directory                       module=cli/local dir=/root/.infrakit/cli fn=github.com/docker/infrakit/pkg/cli/local.Setup
panic: mkdir /root/.infrakit/cli: permission denied

goroutine 1 [running]:
main.main()
	/go/src/github.com/docker/infrakit/cmd/cli/main/main.go:29 +0xf9b
--> 6 plugins should be discoverable: FAIL
Expected line count: 6
Actual line count:        0
cleaning up docker images
```

--- a/scripts/e2e-test-docker-containers.sh
+++ b/scripts/e2e-test-docker-containers.sh
@@ -36,11 +36,11 @@ rm -rf $TEST_DIR/tutorial/*
 
 
 # bind mounts
-volumes="-v $TEST_DIR:/root -v $PWD/docs:/root/docs"
+volumes="-v $TEST_DIR:/ikt -v $PWD/docs:/root/docs"
 
 # set the environment variable to use a shorter path so we don't have
 # problems with Docker for Mac.  See https://github.com/docker/docker/issues/23545
-envs="-e INFRAKIT_PLUGINS_DIR=/root/plugins"
+envs=" -e INFRAKIT_HOME=/ikt -e INFRAKIT_PLUGINS_DIR=/ikt/plugins"
 
 log="--log 5"
 
@@ -60,13 +60,13 @@ echo group > $leaderfile
 
 # start up multiple instances of manager -- typically we want multiple SETS of plugins and managers
 # but here for simplicity just start up with multiple managers and one set of plugins
-server manager infrakit-manager $log --name group --proxy-for-group group-stateless os --leader-file /root/leader --store-dir /root/configs
-server manager1 infrakit-manager $log --name group1 --proxy-for-group group-stateless os --leader-file /root/leader --store-dir /root/configs
-server manager2 infrakit-manager $log --name group2 --proxy-for-group group-stateless os --leader-file /root/leader --store-dir /root/configs
+server manager infrakit-manager $log --name group --proxy-for-group group-stateless os --leader-file /ikt/leader --store-dir /ikt/configs
+server manager1 infrakit-manager $log --name group1 --proxy-for-group group-stateless os --leader-file /ikt/leader --store-dir /ikt/configs
+server manager2 infrakit-manager $log --name group2 --proxy-for-group group-stateless os --leader-file /ikt/leader --store-dir /ikt/configs
 
 sleep 5 # wait for leadership detection to run
 
-server instance-file infrakit-instance-file --dir /root/tutorial/ $log
+server instance-file infrakit-instance-file --dir /ikt/tutorial/ $log
 server group-default infrakit-group-default --poll-interval 500ms --name group-stateless $log
 server flavor-vanilla infrakit-flavor-vanilla $log
 

